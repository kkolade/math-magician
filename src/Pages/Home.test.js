import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('Home component renders correctly', () => {
  const component = renderer.create(<Home />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
