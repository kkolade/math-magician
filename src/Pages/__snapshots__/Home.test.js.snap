// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div>
  <h2
    className="headline"
  >
    Welcome to Our Page!
  </h2>
  <article
    className="article"
  >
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat delectus veritatis architecto ratione aspernatur dolores odit quaerat voluptas, quam quis eveniet beatae ea repudiandae vero temporibus illo facilis iure officia ipsa fugiat distinctio voluptates nulla vel? Fugiat quo aliquam illum obcaecati quasi animi esse, repellat doloremque sed et blanditiis ipsam ratione magnam expedita eum? Dolore id minima voluptatum magni consequatur delectus minus molestias nobis maiores. Numquam explicabo ad, pariatur quas ex sapiente sed repellat. Ab quas consequuntur ipsum, cupiditate nisi nulla, mollitia incidunt cumque cum ipsam corporis laudantium sequi architecto voluptatibus quaerat! Eius expedita blanditiis nam fuga! Molestias, distinctio provident?
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Laborum omnis dolor eveniet quas illum odio incidunt porro tenetur numquam iusto atque culpa doloremque soluta, voluptatem illo iure saepe delectus, id facilis suscipit et. Praesentium mollitia modi exercitationem. Optio natus voluptatibus laudantium tempora esse, facilis veniam incidunt nulla dolorem quia officiis dolor itaque aut vitae doloribus nesciunt laborum quidem libero deleniti accusamus dignissimos in. Aspernatur sint corporis iste molestias? Illo provident alias, amet tempora aliquid reiciendis!
    </p>
  </article>
</div>
`;
